documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-rebalance-{context}']
+
+= Rebalance performance tuning overview 
+
+When an xref:con-optimization-proposals-{context}[optimization proposal] is xref:proc-approving-optimization-proposal-{context}[approved], the partition reassignment commands that comprise the proposal are applied to the Kafka cluster by the Cruise Control server. 
+These commands consist of two types of operations:
+
+* Partition movement: Involves transferring the partition replica and its data to a new location. Partition movements can take one of two forms.
+    ** Inter-broker movement: The partition replica is moved to a log directory on a different broker.
+    ** Intra-broker movement: The partition replica is moved to a different log directory on the same broker.
+
+* Leadership movement: This involves switching the leader of the partition's replicas.
+
+Cruise Control batches up these reassignment commands and issues them to the Kafka cluster. 
+The performance of the cluster during the rebalance is affected by the quantities of each type of movement contained in each batch.
+Cruise Control provides several configuration options for tuning a cluster rebalance for optimal performance. 
+You can set these tuning options at either the xref:ref-cruise-control-configuration-{context}[Cruise Control server] or xref:proc-generating-optimization-proposals-{context}[optimization proposal] levels.
+
+The Cruise Control server setting can be set in either:
+
```

Comment:
Is the lead-in sentence before the bullets correct?

`The Cruise Control server setting can be set in either:`

Are both of them related to to the server settings?